terraform/custom_domains/environment_domains/terraform.tf

```diff
@@ -0,0 +1,25 @@
+terraform {
+
+  required_version = "= 1.3.5"
```

Comment:
Can we align the version to the app terraform? https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/main/terraform/aks/terraform.tf#L2

Reply:
This has been moved to another PR, it can be tested and updated there. https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/1447